app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/addedit/handlers/VaultAddEditCardTypeHandlers.kt

```diff
@@ -24,8 +27,9 @@ data class VaultAddEditCardTypeHandlers(
     val onExpirationMonthSelected: (VaultCardExpirationMonth) -> Unit,
     val onExpirationYearTextChange: (String) -> Unit,
     val onSecurityCodeTextChange: (String) -> Unit,
-
-    ) {
```

Comment:
Odd that we have some things like this lying around. Should we try to make a PR soon where we just run the code formatter on the whole project?

Reply:
The formatter does not actually catch this one

Reply:
Hmm, that's odd. Looks like the extra newline throws it off...if you delete that the formatter updates the other pieces. Might still be worth doing a quick check to see if the formatter catches anything else.